osu.Game.Rulesets.Catch/Edit/CatchSelectionHandler.cs

```diff
@@ -52,16 +52,21 @@ public override bool HandleMovement(MoveSelectionEvent<HitObject> moveEvent)
             return true;
         }
 
-        public override bool HandleFlip(Direction direction)
+        public override bool HandleFlip(Direction direction, bool flipOverOrigin)
```

Comment:
I just noticed that if you press the vertical flip keybind then the pattern will flip horizontally. That's not a fault of this PR (was already wrong, this method isn't checking the direction), but given that we're touching this then may as well fix it here?

Reply:
I'll just make it do nothing for now, yeah?